doc/source/notebooks/advanced/varying-noise.ipynb

```diff
@@ -0,0 +1,673 @@
cE8OiOcKOE8JTVN/H7sgnaqKWrdTJ6dF8UVMV6Dvsu3cLg360723kSwtZRx6/gAtpwr6tmL2ATUN6t4ed05Ij3tuX1CQK/vQVGE97emmqw7v2CYNzeNC+CtzcmGB5a0L4BbO9xdUFTdyE+5Cu7I/w/eFYe1HxxEdYqhjA50ISGrrSi2kku18ddnC7uUg949MYhv9veuAjCEy+dK08LcoZ5satlI9HCw4rYJAfx5/AKlm9/S/dE/LPuTz3dnsG3D75RveQtRFNmRXMT6U1rdlqO1Vrf1tmoJjtZybhkfwP70UobELxhUesD2BCls9SJk40MV1DerOZFbiaO1nJpG06S/dMfBTO2y2f6MUhR2loS526HRiObXAhqAl6MVUV722FnKWK2J11ps9RLf2xGVWkNpbbNOC/tXSwzllcO10oktZwuZNUBpW33hyhHeutfekSHejrxz3XCOna/khT/P9Mum7c/jeexNK+WZKyLxdtKPpa1uVGJnNbjkNf1h6aQgPlVd2fXvpw96UmPTWhBfG2s+f9vhfk48MzeS2+ICOZpdwVubknlvSwpf78tiT2oJBzPK2Hi6gJ8SzrNsaypvb04hrbiWO+MDeWh6mO5afDliIZMQJsljZrQHd08M4r4pIcSFKrAfJLphQ5BKBIb5ORmU0njXxCDsLGS8tTml18cakw+2p5FX2cBLVw3V3n96eQ+uUM9k3jAvgxZO+4unoxX/XBBNRb2S1zYk9e5Q0f7Q3Nrh7sC5/Go+253BzTfcguT6bwdlnWKr9i1vAAAgAElEQVQoFjJJJxnavBgvyuqaSczqvFhnayljSrgbn+xMN2hvrCcGX4VykcilEpys5boR731TQrCSS1mR66o7DVoJSl4aVs60U09xShPCO1tSqGpQ8ujMtpPhocwydmlG8MDUEGxagibkUsmg0AN2h4VUqtMOa50y0LPaMpXXbFdU1DXjYmvBgYwy4kJcqWpQ4nSxUZomYs4QT3allDBniCeb1FqtX7DClnP51b18ZRubzxYxZ0ibXnjtyXxG+jvh17JYl1/ViE+HInAwEunZdZhHK/OHefHI9FBWHr3ANwZ6lbaSXFjNS+vOEevvxC3jAjp9fmdysS7171LG3krOPGkivx3pQkbRjf9sRz2psRmIgrg9DlZyrh/jx9NzI3lidgTXjPTB2kKKiEigwpbpke46u7crh3tfUoWhyegmwW2wsSDGiw0GWJ662Fpw39QQtp4rMrpjQHckFVTz1d4sloz2a7M47OE9eDSnHAGRWDO4NQiCwPxhXjw6I4zfDufyyc70rovj1k5vK60d7pZCubCqkbc3J3Mit5Ln50Vp5XuDuE4xlGA3Oz3HsKkRbljJJWw80/XfWnyogmkR7ryyPqn3kKEeuOyKYtBKJP44ql0AcLWz5K74IN5P9yR7+qd6pyfpku+YOvdanpoTydUjfXVfL4oib25KxpMybo8L5EBGWduocRDHOY4PdtEJ0R1t5MT4OOqK4kBXW7JK63r6cqOhVGuQSgQySmopqWkiLsSVQ5nlZvVd7QvuDlaU1DZx1QhvarBhZ3Ix18T6sOpY1/rajlQ3KjmaU6GzvUktqiGpoJqrWrrEFXXNOF0KHa8W26m4EAUHenBReGxmOLOjPXh53Tne2Jhs0Obvkexyrv/sIJYyCe9eP6LLONaUwppLItrZEEZLUimvUxokGTA1A10Qd4WzrQVjAl2IC1EQ5eWAp6PVZSWRMApdJLgNRiQSgRhfw7TFd8YH4m5vyRsbk03epNFoRJ5ddRona7l2ua4XmlRq1p0q4GbpdpO+ro7YWsp4eEYYN471Z+2JfF7fmMSB9NI2zXFrZ7uVoMmoRAm5Z/axbGsqfx7P4/6podw0zr9tEjuI6xRDmRrhxs52rlo2FjKmhruz4XRhJxeKVqK9HXhgWghvbkrm+PkKquqVff47uyyvQgo7S6oblbrtxaWTgrG3kvHyGVfE0XdpH9TD6WnruSKOn6/kMdkfWMmlJGSWMS7IdcD1gL0xKsBZb7s3LlTB8fOV1DWptMl3Rkps642TuZWM8HPSFVZxIQrO5VcxxLvvNjompZ33qLeTNQEuNiioZM2JfGRSbaR3b24Loijy4bY0HpoeqvvY2hP5SASY32I1ti1p8LpO6NFiOzWlm4S7ViQSgY9uGslN4/z5bHcGt36d2GNq4o7kIm75OgFXO0t+vy+uS5/V6kYltpayQa+h7Av3TAri2/3ZelvU5mYwFsT/4/JjwTBv1hvgxWtjIeOxmeEcyalgW5JhvsX95cfE85zIreSFBVEGBXx8dyCbO+OCGKhLkIutBUsnB/PU7AhUGpFv9mfx7pYU3q2/gmXpnixTXssy5bW8uyWFj33fIT3sHpZODub+qSHYtbeLM0ad0g9fbmOjsLOkvE7/vnJNrA+ltU16FrRdfd3z86Moqm7k92MXWLYtjR8NTByFy7QoBpg31EvXZne0kfPgtFDqU3fSePBL7QO6OT2pNSJvb04hWGHLYukemlRqZBJBK6sYQD2gIbR6EreejCaGKlBpRBKzyvF3sSGnvF2n2ISjuYSscsYFu3AgvQwfJ2v8XKxRi+KgsxRq7z06Z4gH25KLuVJ6gO3JRRRUNXDFUE/Wny6gqqHF7q6L39nPiblcEeOJi632otuoVPPrkVwmhrnh1rJYl1VaN+gz59sjkQgEutr22OW0lEl57eqYFo1xBQs+2sv721I5kq1d7qxrUrHpTCFPrTzJ374/Sqi7HSvvm6CTk3Rk/akC5sdcmnZbOjr4icqlEh4IzCfhh38OyMsxS0HclYfqIGkS/A/zIZUIRHo6cMYAl6PrR/sSrLDlrU2GTZn6Q3F1I29tSiY+1JVFI3p3WMkqrUMiCPi7DryPvEwqYXK4Nj3yydkRPDk7gsdnhfO4/A8el//Bk7MjeGzp3UyLdNcvhlsxRp3ST19uY2NvJW+7/wLTIt1R2Fmw8mjPDj+WMilzh3px98QgnpgVzs1dyPW64
```

Comment:
Could this sentence be framed another way?

Suggestion : "In this demo, we will assume that the noise variances are only known in two groups, in contrast with with first demo, where they were known for each datapoint individually"